client/blocks/site/README.md

```diff
@@ -9,13 +9,14 @@ This component displays a Site item using the data structure provided by sites-l
 import Site from 'blocks/site';
 
 render() {
-	return <Site site={ siteObject } indicator={ true } />;
+	return <Site siteId={ siteId } indicator={ true } />;
 }
 ```
 
 #### Props
 
-* `site (object)` - A site object from `sites-list`. It's required.
+* `siteId (number)` - A site ID. Required, unless a `site` object prop is provided (see below).
+* `site (object)` - A site object. Required, unless a `siteId` prop is provided.
```

Comment:
Looking at the docs, it's not clear whether `siteId` or `site` would be treated with higher priority. Should we indicate that somewhere? I know it wouldn't make sense to pass both props at the same time, but just in case 😉